PurchasesTests/Mocks/MockTrialOrIntroPriceEligibilityChecker.swift

```diff
@@ -19,7 +19,8 @@
 class MockTrialOrIntroPriceEligibilityChecker: TrialOrIntroPriceEligibilityChecker {
 
     convenience init() {
-        let systemInfo = try! MockSystemInfo(platformFlavor: "xyz", platformFlavorVersion: "123", finishTransactions: true)
+        let platformInfo = Purchases.PlatformInfo(flavor: "xyz", version: "123")
+        let systemInfo = try! MockSystemInfo(platformInfo: platformInfo, finishTransactions: true)
```

Comment:
Ditto in all these, you might be able to keep these all in one line using `.init`